hummingbot/connector/exchange/loopring/loopring_exchange.pyx

```diff
@@ -266,9 +266,9 @@ cdef class LoopringExchange(ExchangeBase):
             next_id = self._next_order_id
             if force_sync or self._next_order_id.get(token) is None:
                 try:
-                    response = await self.api_request("GET", NEXT_ORDER_ID, params={"accountId": self._loopring_accountid, "sellTokenId": token})
+                    response = await self.api_request("GET", NEXT_ORDER_ID, params={"accountId": self._loopring_accountid, "sellTokenId": token, "maxNext": "true"})
                     next_id = response["orderId"]
-                    self._next_order_id[token] = next_id
+                    self._next_order_id[token] = next_id + 2
```

Comment:
Why do we increment by 2 here? Perhaps this was part of a test?

Reply:
Hi, the api seemed to be returning the current maximum of the storage ids the user had already used, rather than the next available storage id after that.

Reply:
```suggestion
                    self._next_order_id[token] = next_id + 2  # api returns used count rather than next available